clang-tools-extra/clangd/unittests/SymbolCollectorTests.cpp

```diff
@@ -1477,6 +1477,26 @@ TEST_F(SymbolCollectorTest, Documentation) {
                         forCodeCompletion(false))));
 }
 
+TEST_F(SymbolCollectorTest, DocumentationInMain) {
```

Comment:
Another test case that would be useful to have is one where the declaration and definition both have comments, and checks which one is chosen.

Reply:
Hm, do we really want to specify a behavior there? My thinking was that the comment should be at either the declaration or the definition and otherwise one of them gets chosen at random, depending on the parse order. I suppose one could also cook up some elaborated merging scheme, but that seems like overkill.